### DISCUSSION TITLE
Can't trash folders on different volumes due to permission errors

### DISCUSSION THREAD

**Discussion by medmaca** [Byron/trash-rs#100] [General]
I love `trashy`, it is way faster than the python `trash-cli`.  However, when using on a HPC I can't trash files on non `$HOME` volumes as trashy tries to create a top level` ./Trash-*` folder.  However, I don't have permission to create folders at that level of the volume.

In `trash-cli` this is resolvable as it has a `--trash-dir` option allowing the user to choose the trash location.  Is this something that exists for `trashy`?  If not do you have any suggestions?

**Comment by Byron** [comment]
I think this discussion should move to the home of `trashy` itself: https://github.com/oberblastmeister/trashy .

This is a crate `trashy` is using, and I don't think it supports manually setting a trash directory.
